securedrop_client/gui/widgets.py

```diff
@@ -2009,8 +1996,20 @@ def _request_passphrase(self, bad_passphrase: bool = False):
         else:
             self.passphrase_error_message.hide()
 
+    @pyqtSlot(object)
     def _update(self, status):
-        if status == ExportStatus.USB_NOT_CONNECTED.value:
+        logger.debug('updating status... ')
+        # The first time we see a CALLED_PROCESS_ERROR, tell the user to insert the USB device
+        # in case the issue is that the Export VM cannot start due to a USB device being
+        # unavailable for attachment. According to the Qubes docs:
+        #
+        #   "If the device is unavailable (physically missing or sourceVM not started), booting
+        #    the targetVM fails."
+        #
+        # For information, see https://www.qubes-os.org/doc/device-handling
+        if status == ExportStatus.CALLED_PROCESS_ERROR.value:
```

Comment:
nice refactor that demonstrates one of the advantages to using signals - keeps our return code checks together in one function.

Reply:
hmm although: as is this doesn't handle the situation of:

1. first time (i.e., during pre-flight) `CALLED_PROCESS_ERROR` is returned, call `_request_to_insert_usb_device()` so that the user inserts the USB device again
2. on _subsequent_ occurrences of `CALLED_PROCESS_ERROR`, display a "contact admin" message

one way to proceed is to separate `_update` into two methods: one for updating widget status after pre-flight (where we want this different handling of `CALLED_PROCESS_ERROR`), and one for updating widget status after usb exports attempts, when a `CALLED_PROCESS_ERROR` indicates that there is another problem that requires admin intervention.